TTC.Deployment.AmazonWebServices/Deployer.cs

```diff
@@ -22,28 +23,37 @@ public class Deployer
         private readonly AmazonCloudFormationClient _cloudFormationClient;
         private readonly AmazonS3Client _s3Client;
         private readonly AmazonIdentityManagementServiceClient _iamClient;
-        private readonly AwsConfiguration _awsConfiguration;
         private readonly AmazonAutoScalingClient _autoScalingClient;
-        private readonly AmazonSecurityTokenServiceClient _securityTokenServiceClient;
 
-        public Deployer(AwsConfiguration awsConfiguration) {
-            _awsConfiguration = awsConfiguration;
-            _securityTokenServiceClient = new AmazonSecurityTokenServiceClient(_awsConfiguration.AwsEndpoint);
+        private readonly AwsConfiguration _awsConfiguration;
 
+        public Deployer(AwsConfiguration awsConfiguration) {
             // TODO: don't use EnvironmentAWSCredentials, because it doesn't support environment variables!

```

Comment:
I think this line really needs to be scrapped, but dont understand it well enough.